_data/authors.yml

```
@@ -40,3 +40,17 @@ Anita Graser:
     - label: "Twitter"
       icon: "fab fa-fw fa-twitter-square"
       url: "https://twitter.com/underdarkGIS"
+Juanita Gomez:
+  name        : "Juanita Gomez"
+  bio         : "Community Lead, pyOpenSci & Scientific Python"
```

```suggestion
  bio         : "Community Lead, Scientific Python"
```
I checked in on this position. Essentially because we are hiring a full position in this space and this position ends June 15 which is next week i think we should keep the Scientific Python position but remove pyOpenSci so I don't get in legal trouble with hiring and pay and such.